lib/opensearch-aws-sigv4.rb

```diff
@@ -36,6 +104,7 @@ module Aws
     #   puts client.cat.health
     #
     # @attr [Aws::Sigv4::Signer] sigv4_signer Signer used to sign every request
+    # rubocop:disable Lint/MissingSuper
```

Comment:
This is not a cop you want to disable. Even if we subclass the client, call `super`. The signer should be extracted out of the parameters as I mentioned above.